src/components/Rating.jsx

```diff
@@ -1 +1,24 @@
 // implement Rating component here
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Rating extends React.Component {
+  render() {
+    const { rating } = this.props;
+    return (
+      <p className="rating">
+        { rating }
```

Comment:
Uma dica legal, assim ele vai ficar só com um numero com a classe e ainda vai ter um texto de descrição:
```suggestion
       <p> Rating: <span className:"rating">{ rating } </span> </p>
```

Reply:
Show de bola fili! <3